src/libraries/System.Diagnostics.TextWriterTraceListener/src/System.Diagnostics.TextWriterTraceListener.csproj

```diff
@@ -42,6 +42,7 @@
     <Reference Include="System.ComponentModel.Primitives" />
     <Reference Include="System.Console" />
     <Reference Include="System.Collections.NonGeneric" />
+    <Reference Include="System.Collections.Specialized" />
```

Comment:
Why is this dependency needed?

Reply:
Because `DelimitedListTraceListener` makes use of its `Attributes` property now (inherited from `TraceListener`), which is of type `StringDictionary`.